platform/src/main/java/org/stellar/anchor/platform/service/MetricEmitterService.java

```diff
@@ -0,0 +1,64 @@
+package org.stellar.anchor.platform.service;
+
+import io.micrometer.core.instrument.Metrics;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.annotation.PreDestroy;
+import org.stellar.anchor.config.MetricConfig;
+
+import io.micrometer.core.instrument.Tags;
+import org.stellar.anchor.platform.data.JdbcSep31TransactionRepo;
+import org.stellar.anchor.util.Log;
+
+public class MetricEmitterService {
+  private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
+  private final JdbcSep31TransactionRepo sep31TransactionStore;
+  AtomicInteger pendingStellarTxns = new AtomicInteger(0);
+  AtomicInteger pendingCustomerInfoUpdateTxns = new AtomicInteger(0);
+  AtomicInteger pendingSenderTxns = new AtomicInteger(0);
+  AtomicInteger pendingReceiverTxns = new AtomicInteger(0);
+  AtomicInteger pendingExternalTxns = new AtomicInteger(0);
+  AtomicInteger completedTxns = new AtomicInteger(0);
+
+  public MetricEmitterService(JdbcSep31TransactionRepo sep31TransactionRepo) {
+    this.sep31TransactionStore = sep31TransactionRepo;
+    this.executor.scheduleAtFixedRate(new MetricEmitter(), 0, 30, TimeUnit.SECONDS);
```

Comment:
I don't think we need this level of real-time (every 30 seconds). With a huge database, this could become detrimental to performance. 

💡 Can we have the number of seconds configurable through MetricConfig?